### ISSUE TITLE
False "in stock" notifications

### ISSUE THREAD

**Issue by beserkemptyego** [zeldridge/best-buy-sniper#19]
anyone else getting RTX to appear in stock when using the program? shows up in nightly when I watch the code go through. and an error appears on the website: 

This item is not in your cart yet.
Due to high demand, we're having everyone go through one more step before it can be added.

Essentially it's flooding my notifications and getting nothing done.
![in stock](https://user-images.githubusercontent.com/78816376/107406447-74335980-6ad6-11eb-8525-a463a7e38a45.png)

**Comment by perryleros** [comment]
Yes I am getting the same thing. Shows in stock ONLY through the bot and not if you go there manually.
